WalletWasabi.Tests/UnitTests/Userfacing/ParserTests.cs

```diff
@@ -174,10 +174,10 @@ public void BitcoinAddressParserTests()
 			Assert.False(AddressStringParser.TryParseBitcoinAddress(null!, test.network, out _));
 
 			Assert.True(AddressStringParser.TryParseBitcoinAddress(test.address, test.network, out BitcoinUrlBuilder? result));
-			Assert.Equal(test.address, result!.Address.ToString());
+			Assert.Equal(test.address, result!.Address?.ToString());
```

Comment:
Imho it should be like this:

```suggestion
			Assert.Equal(test.address, result!.Address!.ToString());
```

because we do not expect `result.Address` to be `null` or are we? I would apply the change to all other modified lines.

Reply:
BitcoinUrlBuilder needs to be checked.

Reply:
Done. I initially intended to use `!` to ignore the warnings.